### PR TITLE
Add channel logo extension for relative paths only

### PR DESCRIPTION
**Pull request by phunkyfish**
v4.7.0
- Fixed: Add channel logo extension for relative paths only
- Fixed: Logger fix ported from pvr.hts
- Update: Update readme
- Added: Transform UDP/RTP multicast stream URLs to local udpxy URLs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,6 +111,13 @@ Settings realted to Channel Logos.
     - `Prefer M3U` - Use the channel logo from the M3U if available otherwise use the XMLTV logo.
     - `Prefer XMLTV` - Use the channel logo from the XMLTV file if available otherwise use the M3U logo.
 
+### Advanced
+Advanced settings rsuch a multicast relays.
```
Comment (ksooo) [inline README.md]: typo? => "such as"

File: src/iptvsimple/data/Channel.cpp [hunk 2]
```diff
@@ -102,5 +104,22 @@ void Channel::SetIconPathFromTvgLogo(const std::string& tvgLogo, std::string& ch
...
+void Channel::SetStreamURL(const std::string& url)
+{
+  m_streamURL = url;
+
+  if (Settings::GetInstance().TransformMulticastStreamUrls() &&
+      (StringUtils::StartsWith(url, UDP_MULTICAST_PREFIX) || StringUtils::StartsWith(url, UDP_MULTICAST_PREFIX)))
+  {
+    std::string typePath= "/udp/";
+    if (StringUtils::StartsWith(url, "rtp"))
+      typePath = "/rtp/";
```
Comment (ksooo) [inline src/iptvsimple/data/Channel.cpp]: `const std::string typePath = StringUtils::StartsWith(url, "rtp") ? "/rtp/" : "/udp/";` ?

File: pvr.iptvsimple/changelog.txt [hunk 3]
```diff
@@ -1,5 +1,8 @@
 v4.6.2
 - Fixed: Add channel logo extension for relative paths only
+- Fixed: Logger fix ported from pvr.hts
+- Update: update readme
+- Added: Transform UDP/RTP multicast stream URLs to local udpxy URLs
```
Comment (ksooo) [inline pvr.iptvsimple/changelog.txt:5]: As this is new functionality I would bump addon minor version, not just micro.

File: src/iptvsimple/data/Channel.cpp (hunk 2 above)
Comment (phunkyfish) [reply src/iptvsimple/data/Channel.cpp]: Much better ;)

**Comment by phunkyfish** [comment]
Ok, ready.

**Review by ksooo — APPROVED** [review]

**Pull request merged**